pkg/installer/Dockerfile

```diff
@@ -9,14 +36,14 @@ ENV BUILD_PKGS mkinitfs grep patch make coreutils musl-dev gcc g++ perl \
     util-linux-dev flex linux-headers glib-dev libxrandr-dev zlib-dev \
     libusb-dev kmod-dev curl eudev-dev libdrm-dev i2c-tools-dev hwinfo \
     binutils-dev libaio-dev zlib-static libvncserver-dev python3 pixman-dev \
-    py3-setuptools bash perl cmake m4 pkgconf pkgconfig autoconf-archive tpm2-tss-dev \
+    py3-setuptools bash cmake m4 pkgconf pkgconfig autoconf-archive tpm2-tss-dev \
     cryptsetup-dev go lddtree git
 ENV PKGS mtools dosfstools libarchive-tools sgdisk e2fsprogs util-linux \
     squashfs-tools coreutils tar dmidecode smartmontools libaio libaio-dev \
-    perl glib zlib libusb curl xz pciutils usbutils hdparm util-linux \
-    wireless-tools libxrandr eudev-libs libusb sudo fio iperf3 sysstat \
-    lm-sensors acpi iw libdrm hwinfo dhclient dmidecode smartmontools \
-    pciutils libgcc pixman glib libvncserver musl-utils dhclient \
+    perl glib zlib libusb curl xz pciutils usbutils hdparm ncurses-dev jq \
+    wireless-tools libxrandr eudev-libs sudo fio iperf3 sysstat \
+    lm-sensors acpi iw libdrm hwinfo dhclient e2fsprogs-extra \
+    libgcc pixman libvncserver musl-utils strace \
```

Comment:
Interesting, what is `strace` needed for during installation?

Reply:
This should go away, good catch.